packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem.tsx

```diff
@@ -144,39 +145,33 @@ const NavItem = (props: NavigationItemProps) => {
             $isRounded={isRounded}
             $typographyStyle={typographyStyle}
         >
-            <Icon name={icon} size={iconSize} color={theme.iconSubdued} pointerEvents="none" />
-            <ExpandedSidebarOnly>
```

Comment:
I removed the components that needed the context and replaced with passed prop for better reusability.